packs/equipment-effects/effect-bombers-eye-elixir.json

```diff
@@ -22,31 +22,50 @@
         },
         "rules": [
             {
-                "actorFlag": true,
                 "adjustName": false,
                 "choices": [
                     {
-                        "label": "PF2E.SpecificRule.Equipment.BombersEyeElixir.Lesser",
                         "predicate": [
                             {
                                 "not": "parent:origin:item:slug:bombers-eye-elixir-greater"
                             }
                         ],
-                        "value": 1
+                        "value": "Compendium.pf2e.equipment-srd.Item.Bomber's Eye Elixir (Lesser)"
                     },
                     {
-                        "label": "PF2E.SpecificRule.Equipment.BombersEyeElixir.Greater",
                         "predicate": [
                             {
                                 "not": "parent:origin:item:slug:bombers-eye-elixir-lesser"
                             }
                         ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Bomber's Eye Elixir (Greater)"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "bombers-eye-elixir"
+            },
+            {
+                "actorFlag": true,
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.Numbers.One",
+                        "predicate": [
+                            "bombers-eye-elixir:bombers-eye-elixir-lesser"
+                        ],
+                        "value": 1
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Numbers.Two",
+                        "predicate": [
+                            "bombers-eye-elixir:bombers-eye-elixir-greater"
+                        ],
                         "value": 2
                     }
```

Comment:
In case it's not entirely clear, this flag is needed for an Ephemeral Effect this effect grants. This should be automatically picked after the first Choice Set.

Reply:
Would it be cleaner to use an ActiveEffectLike rather than a hidden ChoiceSet?

Reply:
It would have to be two of them, right? But yeah, I could do that instead. Felt better about the ChoiceSet for some reason!

Reply:
Went with the two AELikes. Let me know if that's what you had in mind